mutiny-core/src/lib.rs

```diff
@@ -704,6 +704,38 @@ pub struct MutinyWalletConfig {
     skip_hodl_invoices: bool,
 }
 
+impl MutinyWalletConfig {
+    pub fn new(
+        xprivkey: ExtendedPrivKey,
+        network: Network,
+        esplora: Option<String>,
+        rgs: Option<String>,
+        lsp: Option<String>,
+    ) -> MutinyWalletConfig {
+        MutinyWalletConfig {
+            xprivkey,
+            #[cfg(target_arch = "wasm32")]
+            websocket_proxy_addr: None,
+            network,
+            user_esplora_url: esplora,
+            user_rgs_url: rgs,
+            lsp_url: lsp,
+            lsp_token: None,
+            lsp_connection_string: None,
+            auth_client: None,
+            subscription_url: None,
+            scorer_url: None,
+            primal_url: None,
+            blind_auth_url: None,
+            hermes_url: None,
+            do_not_connect_peers: false,
+            skip_device_lock: false,
+            safe_mode: false,
+            skip_hodl_invoices: true,
+        }
+    }
+}
```

Comment:
you should just be able to use `MutinyWalletConfigBuilder` in `main.rs` instead of needing to create this